### ISSUE TITLE
Release 0.38.1 mistagged

### ISSUE THREAD

**Issue by fphilipe** [pinterest/ktlint#855]
## Expected Behavior

I just upgraded to version 0.38.1 in order to get the fix introduced in https://github.com/pinterest/ktlint/commit/df3020e234cca8fd9c314fbfcda5d66b54740533. Notice how the commit should be present in 0.38.1:

<img width="689" alt="Screen_Shot_2020-08-25_at_14_24_07" src="https://user-images.githubusercontent.com/128151/91173813-b01c6300-e6de-11ea-80c2-a5bb2ad74c24.png">

Thus, I expected this code to no longer be an offense:

```kotlin
val str = """Hello"""
```

## Observed Behavior

It is still reporting an offense before the string.

It seems that the release was mistagged. The tag is pointing to the wrong commit. There is a branch named `0.38.1`, maybe that one should have been tagged with `0.38.1`:

<img width="1082" alt="Screen_Shot_2020-08-25_at_14_27_00" src="https://user-images.githubusercontent.com/128151/91175168-a72c9100-e6e0-11ea-9682-5c7928466871.png">

The CHANGELOG for v0.38.1 does not include the CHANGELOG entry added by df3020e234cca8fd9c314fbfcda5d66b54740533.

Thus, I suspect that the release 0.38.1 is for e020a05, not df3020e.

## Steps to Reproduce

- Download latest version
- Run ktlint

```sh
curl -sSLO https://github.com/pinterest/ktlint/releases/download/0.38.1/ktlint
chmod a+x ktlint
./ktlint --stdin <<EOF
val str = """hello"""
EOF
```

This will print:

```
<stdin>:1:10: Missing newline after "="
```

**Comment by shashachu** [comment]
@fphilipe this is almost certainly my fault; I went back and forth about it but I ended up actually created a separate branch for the 0.38.1 release which had only the `apiLevel=3` commit cherry-picked in since I didn't want to introduce a behavior change in what was essentially a hotfix. 

So if you want you should be able to verify the fix in 0.39.0-SNAPSHOT, or we'll be releasing 0.39.0 within a couple of weeks. Sorry for the confusion; I'll see if I can fix up the tags.

**Comment by shashachu** [comment]
Tags should be corrected. Apologies for the confusion.

**Comment by fphilipe** [comment]
Thanks @shashachu and no worries 😊 Looking forward to df3020e 😄
